doc/reference/tooling/tcm/tcm_releases/tcm_1.0.rst

```diff
@@ -0,0 +1,82 @@
+.. _tcm_releases_1_0:
+
+Tarantool Cluster Manager 1.0
+=============================
+
+Release date: December 26, 2023
+
+Releases in series: 1.0.4, 1.0.3, 1.0.2, 1.0.1, 1.0.0
```

Comment:
Not sure about listing all the minor releases here. Maybe, smth like `Latest release: 1.0.4` would be enough.